cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -304,7 +758,7 @@ Prepending a query with `CYPHER 3.5` will execute the query with the version of
 
 Below is an example using the older parameter syntax `+{param}+`:
 
-[source, cypher]
+[source, cypher, role="nocopy,norun"]
```

Comment:
Did this work? Louise build seems to still have the copy and run buttons

Reply:
(the build is from before review fixes)

Reply:
It should work, if not lets card it as a bug.